tests/unit/contracts/graph/test_nodes_parsed.py

```diff
@@ -1800,14 +1703,11 @@ def test_timestamp_snapshot_ok(
 def test_check_snapshot_ok(
     basic_check_snapshot_dict,
     basic_check_snapshot_object,
-    basic_intermediate_check_snapshot_object,
 ):
     node_dict = basic_check_snapshot_dict
     node = basic_check_snapshot_object
-    inter = basic_intermediate_check_snapshot_object
 
     assert_symmetric(node, node_dict, SnapshotNode)
-    assert SnapshotNode.from_dict(inter.to_dict(omit_none=True)) == node
```

Comment:
why is this no longer necessary?

Reply:
It was converting from an IntermediateSnapshotNode to a SnapshotNode. The original point of the IntermediateSnapshotNode was that it wouldn't require the validations that we do on a SnapshotNode (but I think that actually broke quite a while back).  I recently did the ticket to move validation of SnapshotNodes to a post-parsing step. So we definitely don't need the IntermediateSnapshotNode anymore.